ebu_tt_live/ui/assets/css/main.css

```diff
@@ -10,6 +10,57 @@
 
 }
 
+body {
+    padding-top: 5px;
+}
+
+#header {
+    padding: 10px;
+    background: rgb(255,255,255); /* Old browsers */
+    background: -moz-linear-gradient(top, rgba(255,255,255,1) 0%, rgba(237,238,244,1) 6%, rgba(210,218,224,1) 49%, rgba(242,244,247,1) 96%, rgba(255,255,255,1) 100%); /* FF3.6-15 */
+    background: -webkit-linear-gradient(top, rgba(255,255,255,1) 0%,rgba(237,238,244,1) 6%,rgba(210,218,224,1) 49%,rgba(242,244,247,1) 96%,rgba(255,255,255,1) 100%); /* Chrome10-25,Safari5.1-6 */
+    background: linear-gradient(to bottom, rgba(255,255,255,1) 0%,rgba(237,238,244,1) 6%,rgba(210,218,224,1) 49%,rgba(242,244,247,1) 96%,rgba(255,255,255,1) 100%); /* W3C, IE10+, FF16+, Chrome26+, Opera12+, Safari7+ */
```

Comment:
because a button used to be defying anything else you may want to do with it so it was standard practice to go with a div that you can control instead of a button that the US replaces with something hideous and possibly dysfunctional. It may have changed but my encounter with browser hell just today makes me want to use a div... and they are first and foremost a list item and only secondary option clickable.. but this is turning into a UX religious debate at this point so I just stop here....